core/store/migrate/migrations/0112_log_poller.sql

```diff
@@ -0,0 +1,32 @@
+-- +goose Up
+CREATE TABLE logs (
+  evm_chain_id numeric(78,0) NOT NULL REFERENCES evm_chains (id) DEFERRABLE,
+  log_index bigint NOT NULL,
+  block_hash bytea NOT NULL,
+  block_number bigint NOT NULL,
+  address bytea NOT NULL,
+  event_sig bytea NOT NULL,
+  topics bytea[] NOT NULL,
+  tx_hash bytea NOT NULL,
+  data bytea NOT NULL,
+  created_at timestamptz NOT NULL,
+  PRIMARY KEY (block_hash, log_index, evm_chain_id)
+);
+
+-- Hot path query - clients searching for their logs.
+CREATE INDEX logs_idx ON logs(evm_chain_id, block_number, address, topics);
```

Comment:
Is this index for the query in `SelectLogsByBlockRangeFilter`? If so, should it be on `event_sig` rather than `topics`?
```suggestion
CREATE INDEX logs_idx ON logs(evm_chain_id, block_number, address, event_sig);
```

Reply:
yep good catch. Changed to using event sig for queries and didn't update the index